jumpscale/tools/redis/redis.py

```diff
@@ -0,0 +1,41 @@
+from jumpscale.god import j
+from jumpscale.core.base import Base, fields
+
+
+class RedisServer(Base):
+    name = fields.String(default="redis")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.cmd = j.tools.startupcmd.get(self.name)
+
+    @property
+    def installed(self):
```

Comment:
this will only work on ubuntu no? i think the sane check would be for redis-server binary in path.
I think we probably should add it for installation instruction on the host, or that's used in case of container only?